docs/pages/reference/access-controls/authentication.mdx

```diff
@@ -267,3 +267,50 @@ See [GitHub OAuth 2.0](../../admin-guides/access-controls/sso/github-sso.mdx) fo
 
 </TabItem>
 </Tabs>
+
+## Require Displaying Message of the Day (MOTD)
+
+Teleport can display a custom MOTD for users prior to authenticating
+in the Teleport Web UI and CLI. 
+
+<Tabs>
```

Comment:
I would use H3-level headings to organize this information instead of Tabs, as I can see a lot of our users reading both tabs.